### PR TITLE
Fromonno

### PR DESCRIPTION
**Pull request by patrickfuhrmann**

### CODE REVIEW

File: config-nfs.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ matching clients can access this export only in read-only mode
 matching clients can access this export only in read-write mode
 
 **noacl**  
-dCache ACLs will be ignored; only posix access permissions will be considered. This is the default.
+dCache ACL's will be ignored; only posix access permissions will be considered. This is the default.
```
Comment (kofemann) [inline config-nfs.md]: Looks good!

Comment (onnozweers) [inline config-nfs.md]: Actually, the Oxford Dictionaries appear to disagree. https://en.oxforddictionaries.com/punctuation/apostrophe , see the MPs example.

It appears that even in German it should be ACLs. https://de.wikipedia.org/wiki/Apostroph , see the CDs example.

Comment (kofemann) [inline config-nfs.md]: Dear @onnozweers, this changes as well as the pull request is just an exercise  to on git workflow and review process in the github. Sorry for confusion!

Comment (onnozweers) [inline config-nfs.md]: Patrick explained it to me too. Good luck with teaching Git! ;-)

**Pull request closed without merging**